FairBid/mintegral.plist.xml

```diff
@@ -64,9 +56,9 @@
                 <key>SKAdNetworkIdentifier</key>
                 <string>y45688jllp.skadnetwork</string>
             </dict>
-            <dict>
+             <dict>
                 <key>SKAdNetworkIdentifier</key>
-                <string>YCLNXRL5PM.skadnetwork</string>
+                <string>yclnxrl5pm.skadnetwork</string>
```

Comment:
this id was already present. but its ok, we are removing duplicates during report generation